### ISSUE TITLE
Achieve this repository to avoid confusion 

### ISSUE THREAD

**Issue by maxspahn** [INTERACT-tud-amr/kinova_ros_interface#3]
As the logic from this repository is now in https://github.com/INTERACT-tud-amr/dinova, this repository should be achieved to avoid confusion.

For now, I simply add a note in the README.md, but when the issue #2 is done (or transferred) it should be achieved.

**Comment by TomasMerva** [comment]
Same issue with [https://github.com/INTERACT-tud-amr/dingo_kinova_description](url), since it is part of [https://github.com/INTERACT-tud-amr/dinova](url)

**Comment by maxspahn** [comment]
Yeah, I added the warning in the dingo_kinova_description already. 
Good catch!

**Comment by TomasMerva** [comment]
I think we could remove `kinova_ros_interface` and  `dingo_kinova_description` repos, since we have already tested `dinova` repo
